mlir/utils/tuna/tuna-script.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
```

Comment:
New file, to manage the Tuna sequence.

A stable persistent Tuna setup has a database running, Tuna installed, and database tables initialised.  At the moment, we don't have a persistent database, and thus have to do all the setup on each run.

Each run then imports configs, starts a session, sets up jobs, executes those jobs, and exports the results to a .tsv file for later perfRunner.py use.